style.css

```diff
@@ -121,8 +112,235 @@ nav {
   margin: 0 12px;
 }
 
-/* Scroll button */
 .scroll-button {
   margin-top: 83px;
   align-items: center;
+  margin-bottom: 63px;
+}
+
+/* Work section */
+.work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  height: 100vh;
+  margin: 10px 0 0 5px;
+}
+
+/* Project title work */
+.title h1 {
+  color: var(--headline-color-1);
+  height: 97px;
+  font-weight: 700;
+  line-height: 80px;
+  font-size: 40px;
+  margin-top: 63px;
+  margin-bottom: 24px;
+}
+
+.indicator {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 62px;
+}
+
+/* Grid container work */
+.project-grid {
+  height: 474px;
+  width: 327px;
+  border-radius: 8px;
+  margin: 24px 0 0 5px;
```

Comment:
- [ ] I would suggest that you display the projects in the work section using the display: grid property. Apply the display: grid to the main container of the work section. This is to make sure you learn how to use the CSS grid to place elements on a page.